omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PersonResource1_8.java

```diff
@@ -1,555 +0,0 @@
-/**
- * This Source Code Form is subject to the terms of the Mozilla Public License,
- * v. 2.0. If a copy of the MPL was not distributed with this file, You can
- * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
- * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
- *
- * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
```

Comment:
Why did you delete this file and others?

Reply:
For the most part they weren't part of the ticket requirements and thus making changes for them would have not been needed so I thought deleting would be an option.

Reply:
For the most part they weren't part of the ticket requirements and thus making changes for them would have not been needed so I thought deleting would be an option.

Reply:
Deleting a file means deleting code. Was that your intention?

Reply:
Oops!! No not at all my intention, I think that's the reason for the travis failure